pandas/tests/frame/test_analytics.py

```diff
@@ -588,6 +588,14 @@ def test_describe_categorical(self):
         result = df3.describe()
         tm.assert_numpy_array_equal(result["cat"].values, result["s"].values)
 
+        # Ensure the index of an an empty categoric DataFrame column
```

Comment:
can you make this a new test

Reply:
also pls add the issue number as a comment

Reply:
Sure, will do

Reply:
Done - let me know if anything else needs doing